backend/src/utils.ts

```diff
@@ -298,8 +299,8 @@ export const tEmbed = t.type({
     t.type({
       name: t.string,
       url: tNullable(t.string),
-      width: tNullable(t.number),
-      height: tNullable(t.number),
```

Comment:
Despite the fact that we should not allow setting these, as they are ignored by Discord's API, we cannot remove them from the schema as that would break existing configurations that are defining these properties. This also applies to the schema below

For clarity, I'm only talking about the removal of `width` and `height`.

Reply:
Additionally, setting proxy URLs should also not be supported as that will also be ignored by Discord.